### PR TITLE
Update for new tool versions

### PR DESCRIPTION
**Pull request by unguiculus**
Signed-off-by: Reinhard Nägele <unguiculus@gmail.com>

### CODE REVIEW

**Comment by unguiculus** [comment]
Actions will yet have to be released. The PR should be updated once this has happened.

File: .github/workflows/lint-test.yaml [hunk 1]
```diff
@@ -11,19 +11,35 @@ jobs:
...
 
-      - name: Run chart-testing (lint)
-        id: lint
-        uses: helm/chart-testing-action@v1.1.0
+      - name: Set up Helm
+        uses: azure/setup-helm@v1
+        with:
+          version: v3.4.0
+
+      - name: Set up Python
```
Comment (davidkarlsen) [inline .github/workflows/lint-test.yaml:21]: maybe just add a comment above why it's required, as before it was abstracted by having it in the docker image - now the user needs to be aware

**Review by davidkarlsen — APPROVED** [review]

**Comment by davidkarlsen** [comment]
just a nit for a comment - else it's fine 👍

File: .github/workflows/lint-test.yaml (hunk 1 above)
Comment (unguiculus) [reply .github/workflows/lint-test.yaml:21]: I added it to the action's readme but I think it makes sense to add a comment here as well.
See https://github.com/helm/chart-testing-action/blob/master/README.md#L24-L26

Comment (unguiculus) [reply .github/workflows/lint-test.yaml:21]: Added a comment

File: .github/workflows/lint-test.yaml [hunk 2]
```diff
@@ -11,19 +11,37 @@ jobs:
...
+
+      # Python is required because `ct lint` runs Yamale (https://github.com/23andMe/Yamale) and
+      # yamllint (https://github.com/adrienverge/yamllint) which require Python
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.7
+
+      - name: Set up chart-testing
+        uses: unguiculus/chart-testing-action@master
```
Comment (scottrigby) [inline .github/workflows/lint-test.yaml]: Now that this has passed CI, do you want to cut a new release for helm/chart-testing-action/releases and change this line to match?

File: .github/workflows/release.yaml [hunk 3]
```diff
@@ -19,19 +19,37 @@ jobs:
...
 
       - name: Add dependency chart repos
         run: |
-          helm repo add stable https://kubernetes-charts.storage.googleapis.com/
-          helm repo add incubator https://kubernetes-charts-incubator.storage.googleapis.com/
+          helm repo add bitnami https://charts.bitnami.com/bitnami
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.0.0
+        uses: helm/chart-releaser-action@master
```
Comment (scottrigby) [inline .github/workflows/release.yaml:50]: To be clear, our strategy here is to merge this PR as a way of testing `helm/chart-releaser-action@master`, and if it releases properly on merge, make a new release for `helm/chart-releaser-action` and do a fast-follow here to update to that release. Right?

Comment (unguiculus) [reply .github/workflows/release.yaml:50]: Good question. How have you done this so far? I think it makes sense to merge first and release right thereafter.

File: .github/workflows/lint-test.yaml (hunk 2 above)
Comment (unguiculus) [reply .github/workflows/lint-test.yaml]: This should, of course, be `helm/chart-testing-action@master`.

File: .github/workflows/release.yaml (hunk 3 above)
Comment (scottrigby) [reply .github/workflows/release.yaml:50]: Yes same. We never wrote this down in any kind of processes doc, so just wanted to be transparent 👍

**Review by scottrigby — APPROVED** [review]

**Pull request merged**